components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/OAuth2ServiceTest.java

```diff
@@ -48,8 +70,17 @@
 /**
  * This class tests the OAuth2Service class.
  */
-@PrepareForTest({OAuth2Util.class, AuthorizationHandlerManager.class, OAuth2Service.class, IdentityTenantUtil.class,
-        OAuthServerConfiguration.class})
+@PrepareForTest({
+        OAuth2Util.class,
+        AuthorizationHandlerManager.class,
+        OAuth2Service.class,
+        IdentityTenantUtil.class,
+        OAuthServerConfiguration.class,
+        AccessTokenIssuer.class,
+        OAuthComponentServiceHolder.class,
+        OAuthUtil.class,
+        OAuthCache.class
+})
 public class OAuth2ServiceTest extends PowerMockTestCase {
```

Comment:
Even though OAuth2ServiceTest is added under "debug" mode in testng file [L84](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/584/files#diff-c2eea2c0360d25f0ace97b92033bdf96R84), you didn't extends "PowerMockIdentityBaseTest"

Reply:
fixed